riotwatcher/_apis/DataDragonApi.py

```diff
@@ -0,0 +1,54 @@
+class DataDragonApi:
```

Comment:
By hardcoding the versions the versions in the library? I guess we could do that but I worry users would end up with old data without realising why.

Reply:
i meant using the version endpoint and caching the latest version.

Reply:
Uhm so we would cache the answer from the version endpoint for a specific region and then have the user pass the region instead of a version? Unfortunately I think something needs to be passed since the regions can be on different versions and also each region can have different endpoints on different versions. For example ATM champions is on 8.16.1 but runes is on 7.23.1.

Reply:
This has been merged, but I believe @duiker101 is right. I think to make the version optional, you'd have to cache the 'realms/{region}.json' file for the region as well, which contains versions for different endpoints, but that means you'd have to do it for each 'new region request'.